crates/belly_core/src/ess/property/parse.rs

```diff
@@ -45,7 +45,7 @@ pub fn val(prop: &StyleProperty) -> Result<Val, ElementsError> {
 /// <!-- @property-type=$val -->
 /// Size type representing `bevy::prelude::Val` type. Possible values:
 /// - `auto` for `Val::Auto`
-/// - `undefined` for `Val::Undefined`
+/// - `undefined` for `Val::Px(0.)`
```

Comment:
undefined has just been replaced with Px(0.), or atleast thats what the migration gided said todo. the v variants seem to be used when refering to the window as the parent and not the node up 1 up in the chain so it will act as if the paranet fills the screen and it said it is in percent.
for exampel on a fullscreen window on a 1080 x 1920 screen:
Vw(10.) = 108.
Vh(10.) = 192.
Vmin(10.) = 108.
vmax(10.) = 192.

I beleve this is done to give garter control to things like popups, since they can keep there parent but use the window size rather then there paranet needing to be 100% x 100%